docs/docs/persistent-volumes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Stateful Applications Using Local Persistent Volumes
+---
+
+Marathon applications lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can create a stateful application by specifying a local persistent volume.
+
+When you specify a local volume or volumes, tasks and their associated data are “pinned” to the node they are first launched on and will be relaunched on that node if they terminate. Local volumes enable stateful tasks because the task can be restarted without data loss.
+
+## Scaling your app
+
+When you scale your app instances/tasks down, the volumes associated with the destroyed tasks/app instances are detached. At this point, you may destroy the volume from the Marathon web interface. You may wish to leave it in the Detached state, however, if you think you will be scaling up your app again; the data on the volume will still be there. **Note:** If your app is deleted, any associated volumes will also be deleted.
+
+## Create an Application with Persistent Local Volumes
+
+1. Create a new Marathon application via the web interface.
+1. Click the Volumes tab.
+1. Choose the size of the volume or volumes you will use. Be sure that you choose a volume size that will fit the needs of your application; you will not be able to modify this size after you launch your application. <!-- note, I need to actually do this and see how you can choose more than one volume -->
+1. Specify the container path, from which your application will read and write data. The container path must be non-nested, e.g. `/var/lib/mysql`, but not  `../../../etc/opt`.
+1. Click Create.
+
+**Note:** After you launch your application, you will not be able to change the specifications for your local persistent volumes.
+
+<!--
+<screen shot>
+-->
+
+Alternatively, you can configure your application to use persistent local volumes in your application definition `config.json` file:
+
+```
+{
+  "id": "/sesame/samson",
+  "residency": {
+    "relaunchEscalationTimeoutSeconds": 30,
+    "taskLostBehavior": "relaunchAfterTimeout"
+  },
+  "cpus": 0.5,
+  "mem": 512.0,
+  "disk": 1024.0, // sandbox disk space
+  "instances": 2,
+  "container": {
+    "type": "MESOS",
+    "volumes": [
+      { // --> This will create a persistent volume
+        "containerPath": "/local/a",
```

Comment:
For the time being, please change containerPath to just `"local"` or `"data"` since we can't have nested/absolute paths